libs/pages/services/src/lib/feature/page-job-create-feature/configure-feature/configure-feature.tsx

```diff
@@ -0,0 +1,114 @@
+import { useEffect } from 'react'
+import { FormProvider, useForm } from 'react-hook-form'
+import { useNavigate, useParams } from 'react-router-dom'
+import { JobConfigureData } from '@qovery/shared/interfaces'
+import {
+  SERVICES_JOB_CREATION_GENERAL_URL,
+  SERVICES_JOB_CREATION_RESOURCES_URL,
+  SERVICES_URL,
+} from '@qovery/shared/router'
+import { toastError } from '@qovery/shared/toast'
+import { FunnelFlowBody, FunnelFlowHelpCard } from '@qovery/shared/ui'
+import { useDocumentTitle } from '@qovery/shared/utils'
+import Configure from '../../../ui/page-job-create/configure/configure'
+import { useJobContainerCreateContext } from '../page-job-create-feature'
+
+export function ConfigureFeature() {
```

Comment:
You don't have the same wording with the `page-database-create-feature` and the `page-application-create-feature` please harmonize for all files thanks 💯

Reply:
I'll do this simplification renamming for page-database and page-application in a coming PR. All these components will be name : StepXXFeature and StepXX